app/enzyme.js

```diff
@@ -0,0 +1,3 @@
+const Adapter = require("enzyme-adapter-react-16");
+
+require("enzyme").configure({ adapter: new Adapter() });
```

Comment:
Is es5 required here? How about: 
```javascript
import Adapter from "enzyme-adapter-react-16" 
import Enzyme from "enzyme" 

Enzyme.configure({ adapter: new Adapter() });
```
Also any specific reason for `.js` and not `.ts` file? 😄

Reply:
Thank you for your attention, it's actually an unnecessary file, as I added later the configuration of `enzyme` is in `app/src/setupTests.js`. I also changed it to `setupTests.ts` :)